codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/proto/example-action.proto

```diff
@@ -27,9 +27,9 @@ service MyService {
   };
 
   rpc simpleMethod(MyRequest) returns (org.external.Empty) {}
-  rpc streamedOutputMethod(MyRequest) returns (stream google.protobuf.Empty) {}
-  rpc streamedInputMethod(stream MyRequest) returns (google.protobuf.Empty) {}
-  rpc fullStreamedMethod(stream MyRequest) returns (stream google.protobuf.Empty) {}
```

Comment:
This is causing a compilation error. Akka gRPC will generate a service two serializers using the same name. 
I reported it here: 
https://github.com/akka/akka-grpc/issues/1526